### PR TITLE
docs: add `README.md`

### PR DESCRIPTION
**Pull request by Dox-Dev**
This PR is dedicated to adding a `README.md` file for the repository, and for saying hi to Mr. Sebastian Luis Ortiz. Hi Basti!

### CODE REVIEW

**Comment by Dox-Dev** [comment]
I have implemented my name into the "Attendees" section. Still though, hi Basti!

File: README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-
----
-## Attendees
-
----
-## Partners
-This workshop is co-hosted by [Secuna](https://secuna.io), in partnership with UP CURSOR and UP ACM.
+# git-workshop-2023
+
+hi
```
Comment (BastiDood) [inline README.md:3]: but y tho?

**Review by BastiDood — CHANGES_REQUESTED** [review]
Thank you, Ellis. But we're gonna need to remove one part here. 🤣

File: README.md (hunk 1 above)
Comment (BastiDood) [reply README.md:3]: If ya don't mind, can you also resolve the merge conflicts by syncing your fork's `main` branch, and then merging `main` _into_ `hi-basti`? Let me know if you need help with that.

Comment (Dox-Dev) [reply README.md]: Sure, lemme go ahead with that.

**Review by Dox-Dev — COMMENTED** [review]
This change was noted, and has been subsequently dealt with in the latest commit.

**Comment by BastiDood** [comment]
Alright, I'll be waiting on that merge conflict resolution. 👌

**Review by Dox-Dev — COMMENTED** [review]
I had forgotten in my previous commit to review the requested changes you've given for my PR, so here you go.

**Review by BastiDood — APPROVED** [review]
Great job on resolving the conflicts. LGTM! 🚀

**Pull request merged**